Sources/Utils/RequestExportOption.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  RequestExportOption.swift
+//  Wormholy-iOS
+//
+//  Created by Daniele Saitta on 15/02/2020.
+//  Copyright © 2020 Wormholy. All rights reserved.
+//
+
+import Foundation
+
+enum RequestExportOption {
+    case flat
+    case curl
+    case postman
```

Comment:
It would be great to add small comments next to each case, so as to better understand what they do (I foresee that this enum could grow in the future)

Reply:
Ok, I'll add some comments. And yes... I'd like to add other representations of request and response in the future (e.g. Charles session)

Reply:
Sounds good!